packages/nuxt/src/head/runtime/composables.ts

```diff
@@ -1,39 +1,57 @@
-import type { HeadEntryOptions, UseHeadInput, ActiveHeadEntry } from '@vueuse/head'
-import { useSeoMeta as _useSeoMeta } from '@vueuse/head'
+import type { HeadEntryOptions, UseHeadInput, ActiveHeadEntry } from '@unhead/vue'
+import { useServerHeadSafe as _useServerHeadSafe, useServerSeoMeta as _useServerSeoMeta, useHead as _useHead, useServerHead as _useServerHead } from '@unhead/vue'
 import type { HeadAugmentations } from 'nuxt/schema'
-import { useNuxtApp } from '#app/nuxt'
 
 /**
- * You can pass in a meta object, which has keys corresponding to meta tags:
- * `title`, `base`, `script`, `style`, `meta` and `link`, as well as `htmlAttrs` and `bodyAttrs`.
+ * Allows you to manage your head tags in a programmatic and reactive way.
  *
- * Alternatively, for reactive meta state, you can pass in a function
- * that returns a meta object.
+ * @see https://nuxt.com/docs/getting-started/seo-meta
+ * @see https://unhead.harlanzw.com/guide/composables/use-head
  */
 export function useHead<T extends HeadAugmentations> (input: UseHeadInput<T>, options?: HeadEntryOptions): ActiveHeadEntry<UseHeadInput<T>> | void {
-  return useNuxtApp()._useHead(input, options)
+  return _useHead(input, options)
 }
 
 /**
- * The `useSeoMeta` composable lets you define your site's SEO meta tags
- * as a flat object with full TypeScript support.
+ * The `useServerHead` composable is identical to `useHead` except that it will
+ * have no effect (and will return nothing) if called on the client.
  *
- * This helps you avoid typos and common mistakes, such as using `name`
- * instead of `property`.
+ * @see useHead
+ * @see https://unhead.harlanzw.com/guide/composables/use-server-head
+ */
+export function useServerHead<T extends HeadAugmentations> (input: UseHeadInput<T>, options?: HeadEntryOptions): ActiveHeadEntry<UseHeadInput<T>> | void {
+  if (process.server) {
+    return _useServerHead(input, options)
+  }
+}
+
+/**
+ * The `useServerHeadSafe` composable is identical to `useHeadSafe` except that it will
+ * have no effect (and will return nothing) if called on the client.
  *
- * It is advised to use `useServerSeoMeta` unless you _need_ client-side
- * rendering of your SEO meta tags.
+ * @see useHeadSafe
  */
-export const useSeoMeta: typeof _useSeoMeta = (meta) => {
-  return _useSeoMeta(meta)
+export const useServerHeadSafe: typeof _useServerHeadSafe = (input, options) => {
```

Comment:
Both `useServerHeadSafe` and `useServerSeoMeta` can be imported from @unhead/vue when the Vite plugin is implemented